### PR TITLE
chore: rewrite beta clients

### PR DESCRIPTION
**Pull request by codyoss**
This change rewrites signature for all beta clients to be in terms of the new type definitions. This is safe because all the modules are updated to the latest genproto version which has corresponding type aliases.

Also removed a secretmanager beta client that was not mapping to the correct protos.

Changes:

feat(binaryauthorization): rewrite signatures in terms of new types for betas

feat(oslogin): rewrite signatures in terms of new types for betas

feat(retail): rewrite signatures in terms of new types for betas

feat(dataform): rewrite signatures in terms of new types for betas

feat(datalabeling): rewrite signatures in terms of new types for betas

feat(documentai): rewrite signatures in terms of new types for betas

feat(memcache): rewrite signatures in terms of new types for betas

feat(metastore): rewrite signatures in terms of new types for betas

feat(notebooks): rewrite signatures in terms of new types for betas

feat(phishingprotection): rewrite signatures in terms of new types for betas

feat(recommendationengine): rewrite signatures in terms of new types for betas

feat(securitycenter): rewrite signatures in terms of new types for betas

feat(servicedirectory): rewrite signatures in terms of new types for betas

feat(webrisk): rewrite signatures in terms of new types for betas

feat(dataqna): rewrite signatures in terms of new types for betas

feat(functions): rewrite signatures in terms of new types for betas

feat(lifesciences): rewrite signatures in terms of new types for betas

feat(mediatranslation): rewrite signatures in terms of new types for betas

feat(osconfig): rewrite signatures in terms of new types for betas

feat(security): rewrite signatures in terms of new types for betas

feat(vision): rewrite signatures in terms of new types for betas

feat(artifactregistry): rewrite signatures in terms of new types for betas

feat(automl): rewrite signatures in terms of new types for betas

feat(gkeconnect): rewrite signatures in terms of new types for betas

feat(networkconnectivity): rewrite signatures in terms of new types for betas

feat(redis): rewrite signatures in terms of new types for betas

feat(talent): rewrite signatures in terms of new types for betas

feat(videointelligence): rewrite signatures in terms of new types for betas

feat(assuredworkloads): rewrite signatures in terms of new types for betas

feat(billing): rewrite signatures in terms of new types for betas

feat(dialogflow): rewrite signatures in terms of new types for betas

feat(recaptchaenterprise): rewrite signatures in terms of new types for betas

feat(recommender): rewrite signatures in terms of new types for betas

feat(scheduler): rewrite signatures in terms of new types for betas

feat(cloudtasks): rewrite signatures in terms of new types for betas

feat(workflows): rewrite signatures in terms of new types for betas

feat(analytics): rewrite signatures in terms of new types for betas

feat(area120): rewrite signatures in terms of new types for betas

feat(datacatalog): rewrite signatures in terms of new types for betas

feat(datastream): rewrite signatures in terms of new types for betas

feat(domains): rewrite signatures in terms of new types for betas

feat(gaming): rewrite signatures in terms of new types for betas

feat(gkehub): rewrite signatures in terms of new types for betas

feat(networksecurity): rewrite signatures in terms of new types for betas

feat(speech): rewrite signatures in terms of new types for betas

feat(dataflow): rewrite signatures in terms of new types for betas

feat(language): rewrite signatures in terms of new types for betas

feat(privatecatalog): rewrite signatures in terms of new types for betas

feat(secretmanager): remove beta client that no longer has source protos

### CODE REVIEW

File: dialogflow/cx/apiv3/agents_client.go [hunk 1]
```diff
@@ -25,7 +25,6 @@ import (
 
 	"cloud.google.com/go/longrunning"
 	lroauto "cloud.google.com/go/longrunning/autogen"
-	structpb "github.com/golang/protobuf/ptypes/struct"
```
Comment (quartzmo) [inline dialogflow/cx/apiv3/agents_client.go:28]: Is `dialogflow/cx/apiv3` a beta client?

Comment (codyoss) [reply dialogflow/cx/apiv3/agents_client.go:28]: no, This must be remnants of a recent generator upgrade. If you look at the files edited in this folder they are just using the new proto package that was previously aliased. I think we have seen this exact change surface in other PRs recently too. I am not sure how this got into the changeset, but it is safe.

Comment (quartzmo) [reply dialogflow/cx/apiv3/agents_client.go:28]: Gotcha, thanks!

**Review by quartzmo — APPROVED** [review]
LTGM

**Pull request merged**